automod/commands.go

```diff
@@ -198,14 +198,18 @@ func (p *Plugin) AddCommands() {
 			}
 
 			for name, count := range violations {
-				out += fmt.Sprintf("Violation: %-20s Count: %d\n", name, count)
+				if len([]rune(name)) > 31 {
+					out += fmt.Sprintf("%-31s Count: %d\n", common.CutStringShort(name, 30), count)
+				} else {
+					out += fmt.Sprintf("%-31s Count: %d\n", name, count)
+				}
```

Comment:
The conditional branch seems unnecessary now that we're using `CutStringShort`.

Reply:
You make a fair point sir. I will address when I get back to the desk.